charts/orchestrator/values-rc.yaml

```diff
@@ -0,0 +1,22 @@
+isReleaseCandidate: true # Indicates RC builds should be used by the chart to install RHDH and Sonataflow
+ocpVersion: v4.15
+
+sonataFlowOperator:
+  enabled: true # whether the operator should be deployed by the chart
+  subscription:
+    namespace: openshift-serverless-logic # namespace where the operator should be deployed
+    channel: alpha # channel of an operator package to subscribe to
+    installPlanApproval: Automatic # whether the update should be installed automatically
```

Comment:
What are the possible values?

Reply:
See https://olm.operatorframework.io/docs/concepts/crds/subscription/ (Automaic or Manual).
Perhaps there is Default as well, I recall seeing it, but nothing in the official doc that mentions it.